frontend/src/old-pages/Clusters/Cost.tsx

```diff
@@ -0,0 +1,418 @@
+              Activate Tags
+            </Button>
+            <DateRangePicker
+              onChange={({ detail }) => {
+                fetchGraphData(detail.value);
+                setDateValue(detail.value);
+              }}
+              value={dateValue}
+              relativeOptions={Options}
+              i18nStrings={{
+                todayAriaLabel: "Today",
+                nextMonthAriaLabel: "Next month",
+                previousMonthAriaLabel: "Previous month",
+                customRelativeRangeDurationLabel: "Duration",
+                customRelativeRangeDurationPlaceholder: "Enter duration",
+                customRelativeRangeOptionLabel: "Custom range",
+                customRelativeRangeOptionDescription:
+                  "Set a custom range of days in the past. Use Days as Unit of Time.",
+                customRelativeRangeUnitLabel: "Unit of time",
+                formatRelativeRange: (e) => {
+                  const t = 1 === e.amount ? e.unit : `${e.unit}s`;
+                  return `Last ${e.amount} ${t}`;
+                },
+                formatUnit: (e, t) => (1 === t ? e : `${e}s`),
+                dateTimeConstraintText:
+                  "Select any Range of days. Use 24 hour format.",
+                relativeModeTitle: "Relative range",
+                absoluteModeTitle: "Absolute range",
+                relativeRangeSelectionHeading: "Choose a range",
+                startDateLabel: "Start date",
+                endDateLabel: "End date",
+                startTimeLabel: "Start time",
+                endTimeLabel: "End time",
+                clearButtonLabel: "Clear and dismiss",
+                cancelButtonLabel: "Cancel",
+                applyButtonLabel: "Apply",
+              }}
+              placeholder="Filter by a date range"
+            />
+            <ProgressBar
+              value={budgetUsed}
+              additionalInfo="Create a Budget with an identical name as your respective cluster for accurate information"
+              description="Progress"
+              label="Cost Usage of Budget"
+            />
+            <HelpTooltip>
+              If you haven't created a budget, create a custom cost budget on{" "}
+              <a
+                href="https://us-east-1.console.aws.amazon.com/billing/home?#/budgets/overview"
+                target="_blank"
+              >
+                {" "}
+                AWS Budgets
+              </a>
+              .
+            </HelpTooltip>
+            <Button
+              href={`https://${region}.console.aws.amazon.com/cost-management/home?region=${region}"#/custom?groupBy=InstanceType&hasBlended=false&hasAmortized=false&excludeDiscounts=true&excludeTaggedResources=false&excludeCategorizedResources=false&excludeForecast=false&timeRangeOption=Custom&granularity=Daily&reportName=&reportType=CostUsage&isTemplate=true&filter=${JSON.stringify(
```

Comment:
I'd break this out into a separate variable as it's encoding a bit of logic that would make the view easier to read if factored out.

Reply:
resolved